tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/bala/record/RecordInBalaTest.java

```diff
@@ -45,6 +47,16 @@ public void testRestFieldTypeDefAfterRecordDef() {
         BRunUtil.invoke(result, "testCRRestFieldInCR");
     }
 
+    @Test
+    public void negativeTests() {
+        CompileResult result = BCompileUtil.compile("test-src/record/record_negative_bala.bal");
+        Assert.assertEquals(result.getErrorCount(), 2);
```

Comment:
Let's use a variable for the error index and assert at the end against the error count instead.